sormas-api/src/main/resources/captions.properties

```diff
@@ -520,6 +521,7 @@ contactOnlyQuarantineHelpNeeded=Help needed in quarantine
 contactOnlyWithExtendedQuarantine=Only contacts with extended quarantine
 contactOnlyWithReducedQuarantine=Only contacts with reduced quarantine
 contactOnlyWithSharedEventWithSourceCase=Only contacts with source case linked to the specified event
+contactOnlyWithSourceCaseInEvent=Only contacts whose source case is linked to this event
```

Comment:
Could you elaborate on `should be redundant` ? I'm not sure I get it.

Do you mean that the both of them describe the same thing and therefore we should keep only one ? If so, these aren't used for the same filter.
They look similar thought, but `contactOnlyWithSharedEventWithSourceCase` is for contacts which share an event with their source case and `contactOnlyWithSourceCaseInEvent` is for contacts whose source case is related to the given event.
Even if I gotta admit the description `Only contacts with source case linked to the specified event` kinda is misleading on what the filter actually does but it's what I was told to put in #3314, it apparently is more clear for users in the contact directory search.

Those two names aren't really meaningful either but I tried to keep the existing naming style of contactOnlyWith**** and haven't found better ones.

Reply:
My impression is that both properties basically do the same right now because the new property that you added is used to filter for contacts with source cases in the same event, i.e. they share the same event with the source case. That's why I think the new property is redundant because you should be able to just use the one that is already there and was renamed in #3314.

Reply:
As I said, `contactOnlyWithSharedEventWithSourceCase` filter contacts whose person is linked to the same event their source case is - **and only that**. Whereas `contactOnlyWithSourceCaseInEvent` filter contacts whose source case is related to the given event.

Let's say we have :
- Contact1 (Person1) linked to Event1, and its source case is linked to Event1.
- Contact2 (Person1) linked to Event2, and its source case is linked to Event2.
- Contact3 (Person1) linked to Event3, and its source case is linked to Event4.

In Event1's event participants, we check the detail page of the one whose person is Person1. 
We have our contactListComponent which displays the three contacts by default.

If we tick the "Only contacts whose source case is linked to this event" we'd expect only one result : Contact1.
Yet, if I used the #3314 filter I'd get two results : Contact1 and Contact2.
That's not quite the expected behaviour.

Therefore I don't really get how I could just use the one renamed in #3314 :shrug:


Though, I will rename the captions name `contactOnlyWithSourceCaseInEvent` into `contactOnlyWithSourceCaseInGivenEvent` for consistency's sake, as I renamed the filter's name.